### PR TITLE
Generate a nicer changelog for the scratch task srpm.

### LINKED ISSUE #19: Changelog entry for scratch build is useless
It would be nice if the scratch build can contain some more useful entry. You can compare the-new-hotness changelog entry with previous entries for rubygem-jbuilder build for example [1].

Not mentioning that the formatting with quotes etc is suboptimal.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1154598#c6

### PR DESCRIPTION
**Pull request by ralphbean**
This should fix #19.

### CODE REVIEW

**Comment by ralphbean** [comment]
@voxik, does this look like it will do it for you?

File: hotness/buildsys.py [hunk 1]
```diff
@@ -96,8 +96,8 @@ def handle(self, package, upstream, version, rhbz):
             cmd = [
                 '/usr/bin/rpmdev-bumpspec',
                 '--new', upstream,
-                '-c', '"Latest upstream, %s for #%s"' % (upstream, rhbz),
-                '-u', '"%s"' % self.userstring,
+                '-c', 'Update to %s (#%s)' % (upstream, rhbz.bug_id),
```
Comment (relrod) [inline hotness/buildsys.py]: Should this be `#%d` instead?

Comment (ralphbean) [inline hotness/buildsys.py]: Why, yes it should, my type-oriented friend.

(fwiw, the ``%s`` works.. but ``%d`` would be more explicit).

Comment (relrod) [inline hotness/buildsys.py]: > Why, yes it should, my type-oriented friend.

In a dependently-typed language (Idris, Agda, Coq, etc.) you could detect that mismatch at compile time ;). Not in Haskell though. :P

Comment (relrod) [inline hotness/buildsys.py]: @ralphbean If you want to see how that works (it's really neat): https://www.youtube.com/watch?v=fVBck2Zngjo

**Comment by voxik** [comment]
Looks good if I can parse the code correctly :+1:

**Comment by pypingou** [comment]
:+1: for me as well

**Comment by ralphbean** [comment]
Thanks both!

**Pull request merged**